Makefile

```
@@ -394,6 +399,10 @@ bundle-ocp: manifests kustomize operator-sdk ## Generate bundle manifests and me
 bundle-build: ## Build the bundle image.
 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
+.PHONY: bundle-build-ocp
+bundle-build-ocp: ## Build the bundle image.
+	docker build -f bundle-ocp.Dockerfile -t $(BUNDLE_IMG) .
```

```suggestion
	${CONTAINER_TOOL} build -f bundle-ocp.Dockerfile -t $(BUNDLE_IMG) .
```
Could you also modify the previous usages of `docker`?

---

yes